atom/browser/net/atom_cert_verifier.h

```diff
@@ -25,7 +25,7 @@ struct VerifyRequestParams {
 
 class AtomCertVerifier : public net::CertVerifier {
  public:
-  explicit AtomCertVerifier(AtomCTDelegate* ct_delegate);
+  explicit AtomCertVerifier(std::shared_ptr<AtomCTDelegate> ct_delegate);
```

Comment:
This should be a `const std::shared_ptr<AtomCTDelegate>&` here and in the .cc to avoid unnecessary temporaries

Reply:
:+1: